server/routes/api/v1/adoptablePetsRouter.js

```diff
@@ -14,4 +14,24 @@ adoptablePetsRouter.get("/", async (req, res) => {
   }
 })
 
+adoptablePetsRouter.get("/:type/:id", async (req, res) => {
```

Comment:
If we're trying to view one adoptable pet, do we need to know it's type? If we have the id we can just query for the pet, and use `petType()` to get that information. Looks like the endpoint here is a little more complicated than it needs to be

Reply:
Per our discussion with Kerrin, the more complicated adoptablePetsRouter makes sense with the acceptance criteria for this story.